### DISCUSSION TITLE
Campus Experts

### DISCUSSION THREAD

**Discussion by KirolosMFahem** [community/community#132938] [GitHub Education]
### Hi everyone,

Hi everyone

I've applied for GitHub Campus Experts last February and got rejected without a feedback so I can know what to include next time
I hope one of the GitHub Education team get back to me, hope you can help me make this post reach one them

Regards
Kirolos

**Comment by davevad93** [comment]
Hi @KirolosMFahem ,

I'm not from the GitHub Education/GitHub team, but here are listed the requirements to [Applying to be a GitHub Campus Expert](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/applying-to-be-a-github-campus-expert), there's also a pinned post about the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) if you need that info as well. I also suggest you to add the `GitHub Global Campus` and `Global Campus Benefits` labels to this discussion to make it easier for someone from the Education team to reach out to you.
